client/src/components/Downloads/RecentDownloads.vue

```diff
@@ -0,0 +1,68 @@
+<script setup lang="ts">
+import { computed } from "vue";
+import { useRouter } from "vue-router/composables";
+
+import { useDownloadTracker } from "@/composables/downloadTracker";
+import type { MonitoringRequest } from "@/composables/persistentProgressMonitor";
+import { useRoundRobinSelector } from "@/composables/roundRobinSelector";
+import { DEFAULT_POLL_DELAY } from "@/composables/shortTermStorageMonitor";
+import { useUserStore } from "@/stores/userStore";
+
+import Heading from "@/components/Common/Heading.vue";
+import ListHeader from "@/components/Common/ListHeader.vue";
+import DownloadItemCard from "@/components/Downloads/DownloadItemCard.vue";
+
+const router = useRouter();
+
+const userStore = useUserStore();
+const { downloadMonitoringData, untrackDownloadRequest } = useDownloadTracker();
+
+const isEmpty = computed(() => {
+    return downloadMonitoringData.value.length === 0;
+});
+
+const currentListView = computed(() => userStore.currentListViewPreferences.recentDownloads || "grid");
+
+function onGoTo(to: string): void {
+    router.push(to);
+}
+
+function onDelete(request: MonitoringRequest): void {
+    untrackDownloadRequest(request);
+}
+
+function onDownload(url: string): void {
+    window.open(url, "_blank");
+}
+
+const downloadsInProgress = computed(() => {
+    return downloadMonitoringData.value.filter((data) => !data.isFinal);
+});
+
+const roundRobinSelector = useRoundRobinSelector(downloadsInProgress, DEFAULT_POLL_DELAY);
+
+const taskIdToUpdate = computed(() => {
+    return roundRobinSelector.currentItem.value?.taskId ?? null;
+});
+</script>
+<template>
+    <div class="recent-downloads">
+        <Heading h1 separator inline size="lg">Recent Exports & Downloads</Heading>
```

Comment:
Suggestion: Replace the current `<Heading>` with the existing `BreadcrumbHeading` component for consistency.

Reply:
Something like https://github.com/galaxyproject/galaxy/pull/20585/commits/d5b5ac9cbf6db78641a4683f9d772de34965a689?
I thought we would use the BreadcrumbHeading only when we require drilling down :thinking:
But I guess it is more consistent if we want to do it in the future :+1: 
By the way, will it work for navigating to the items and then back to the downloads? Not sure how to set that up.

Reply:
Right, thank you @davelopez!

Currently, if the item has `to` property, it renders as an active path. #20618 is fixing this.
<img width="503" alt="image" src="https://github.com/user-attachments/assets/1147afd7-f7a5-40c5-811a-0f3f5803031d" />

BreadcrumbHeading is designed to maintain a consistent heading across every page.